packages/components/src/SquiggleChart.tsx

```diff
@@ -42,6 +44,22 @@ export interface SquiggleChartProps {
   onEnvChange?(env: exportEnv): void;
   /** CSS width of the element */
   width?: number;
+  height?: number;
+}
+
+const Error = styled.div`
+  border: 1px solid #792e2e;
+  background: #eee2e2;
+  padding: 0.4em 0.8em;
+`;
+
+const ShowError: React.FC<{ heading:string, errorMessage: string }> = ({ heading="Error", errorMessage }) => {
```

Comment:
Should ShowError here be using the children prop for the error message?

Reply:
Good point, changed!